common/src/main/java/org/apache/uniffle/common/rpc/GrpcServer.java

```diff
@@ -49,6 +49,7 @@ public class GrpcServer implements ServerInterface {
 
   private static final Logger LOG = LoggerFactory.getLogger(GrpcServer.class);
 
+  public static int testVarCount = 0;
```

Comment:
Could we use private here and add a getter for this variable? Could we use `volatitle` here? Because this variable is used for multi threads?  Could we use `boolean` type here? Should we use `static` here? Maybe we should give this variable a better name like `hasExecuted`?